packages/core/core/src/services/entity-service/components.ts

```diff
@@ -22,14 +22,6 @@ type ComponentBody = {
 
 const isDialectMySQL = () => strapi.db?.dialect.client === 'mysql';
 
-function omitComponentData(
```

Comment:
Why was it removed ?

Reply:
Ah I didnt realise this was a TS overload thing tbh, I'll change this back